src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -86,15 +80,22 @@ public CreateCaseCallbackService(
                 ))
                 .mapError(errors -> errors.flatMap(Function.identity()))
                 .flatMap(Function.identity())
-                .toEither()
-                .mapLeft(Seq::asJava)
+                .mapError(Seq::asJava)
             )
-            .peekLeft(warnings ->
-                log.warn("Warnings found during callback process:\n  - {}", String.join("\n  - ", warnings))
-            );
+            .getOrElseGet(errors -> new ProcessResult(emptyList(), errors));
+
+        if (!result.getWarnings().isEmpty()) {
+            log.warn("Warnings found during callback process:\n  - {}", String.join("\n  - ", result.getWarnings()));
+        }
+
+        if (!result.getErrors().isEmpty()) {
+            log.error("Errors found during callback process:\n  - {}", String.join("\n  - ", result.getErrors()));
```

Comment:
could you please log ccd case reference and jurisdiction/service name

Reply:
Can do reference for now. for service will need a further split as it might be one of the reasons callback failed so comes out from validation process/progress

Reply:
oh. actually, ID comes from validation too